www/vonm10/cv01/index.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+$avatar = 'avatar.jpg';
+$name = 'Matyas';
+$surname = 'Vondra';
+$age = 2021 - 2000;
+$position = 'Junior Quality Analyst';
+$company = 'Omio';
+
+$street = 'Fake Street';
+$number = '133/17';
+$psc = '123 45';
+$city = 'Prague';
+
+$phone = '123 456 789';
+$mail = 'vonm10@vse.cz';
+$webpage = 'eso.vse.cz/~vonm10/cv01';
+$availability = 'Not available.';
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Business card</title> 
+<style>
+.firstRow {display : flex; text-align:center; align-items:center; justify-content:center;}
+.secondRow {text-align:center; align-items:center;}
+.card {padding:40px; background-color : #C6C6C6; width:350px}
+</style>
+</head>
+
+<body>
+<div class ="card">
+<div class = "firstRow">
+<div>
+<img src = <?php echo $avatar ?> width ="50" height ="50" alt= "logo">

```

Comment:
vygenerovany HTML neni validni, zvalidujte to na validator.w3.org

Reply:
![image](https://user-images.githubusercontent.com/72431403/109040014-bf9c5900-76cd-11eb-87fa-35851d01ef4d.png)

Reply:
- chybi indentace kodu
- ukoncovaci cast tagu na dalsim radku musi byt indentovana stejne jako zacinajici cast
- hodnoty atributu piseme do uvozovek
- css piseme do samostatneho souboru
![image](https://user-images.githubusercontent.com/20724910/109041383-3e45c600-76cf-11eb-838a-3540b8662533.png)

Reply:
CSS samozrejme predelam, ale s identaci nevim co delat. Ve visual studiu vypada vse spravne (viz screenshot) a hodnoty atributu jsou resene podle zadani pres php

![image](https://user-images.githubusercontent.com/72431403/109042947-0fc8ea80-76d1-11eb-95a4-bf1f37c55bfd.png)

Reply:
- indentace uz jsou dobre udelane
- pridejte jeste uvozovky a bude to
![image](https://user-images.githubusercontent.com/20724910/109047878-ad72e880-76d6-11eb-985c-be155ab396bc.png)

Reply:
Tak snad uz je to v poradku :) Diky